zproject/urls.py

```diff
@@ -498,6 +498,10 @@
     url(r'^json/', include(v1_api_and_json_patterns)),
 ]
 
+# This url serves the number of online users to be set on the README.md
+# community button
```

Comment:
This comment can be "Serves the shields.io badge"

Reply:
Yeah, it's better.